portfolio/pom.xml

```diff
@@ -40,6 +40,14 @@
     </dependency>
   </dependencies>
 
+<!-- gson -->
+ <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+ </dependencies>
```

Comment:
This doesn't have a corresponding opening `<dependencies>` tag, you need to delete the one above on line 41.